internal/manager/manager.go

```diff
@@ -54,6 +53,7 @@ const (
 
 type manager struct {
```

Comment:
I would be suggested to remove this package. And use all this code inline. 
@edwarnicke Thoughts?

Reply:
In the generic I agree... not sure whether its necessary before the release or not though.  How much work would it be and how much risk would it entail?

Reply:
I think you're correct. Let's consider this after release.